### DISCUSSION TITLE
The Story of Yigal Accad

### DISCUSSION THREAD

**Discussion by tomerlichtash** [tomerlichtash/mels-loop#236] [General]
An e-mail sent to [Mel's Loop](https://melsloop.com) editorial has brought to our attention the story of _Yigal Accad_, a young programmer who, in the late 1950's, had worked with _Chaim Leib Pekeris_ of Weizmann Institute to execute his "crazy plan" over the WEIZAC computer.

Here're some initial materials to facilitate the story:

* [The 1958 Pekeris-Accad-WEIZAC Ground-Breaking Collaboration that Computed Ground States of Two-Electron Atoms (and its 2010 Redux) / Christoph Koutschan, Doron Zeilberger](https://arxiv.org/abs/1006.0200)
* Possibly the ad published by Pekeris that Yigal Accad had picked up. Source: [Haaretz, Nov 1st 1957](https://drive.google.com/file/d/15Sgjb6uRTnoRbVxqLph8PUnOZNv30g3e/view?usp=sharing)
* Wikipedia entry for [Haim Leib Pekeris](https://he.wikipedia.org/wiki/%D7%97%D7%99%D7%99%D7%9D_%D7%9C%D7%99%D7%99%D7%91_%D7%A4%D7%A7%D7%A8%D7%99%D7%A1)
* Wikipedia entry for [WEIZAC](https://en.wikipedia.org/wiki/WEIZAC)

**Comment by georgeweigt** [comment]
[Pekeris's 1958 paper referenced by Koutschan and Zeilberger](https://georgeweigt.github.io/Pekeris-1958.pdf)

[Notes on Coding for WEIZAC](https://georgeweigt.github.io/Notes_On_Coding_For_WEIZAC_Sep60.pdf)
